src/amqp-client/channel.cr

```diff
@@ -51,6 +51,7 @@ class AMQP::Client
     end
 
     def close(frame : Frame::Channel::Close) : Nil
+      write Frame::Channel::CloseOk.new(@id)
```

Comment:
Should probably catch any write exception and continue with the close procedure regardless.